react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,34 @@
+import { isArray } from './utils';
+
+export function min(expect, ...c) {
+  if (expect === undefined) {
+    return expect;
+  } else {
+      if (isArray(expect) === false) {
+        return Math.min(expect, ...c);  
+      } else {  
+        const [a, b] = expect;
+        return Math.min(a, b);
```

Comment:
You can return Math.min(...expect). Because you are established the length of the array.